nautobot/dcim/models/power.py

```diff
@@ -175,5 +266,47 @@ def save(self, *args, **kwargs):
     def parent(self):
         return self.power_panel
 
+    @property
+    def occupied_positions(self):
+        """All circuit positions occupied by this feed as comma-separated string."""
+        positions = self.get_occupied_positions()
+        return ", ".join(map(str, sorted(positions))) if positions else ""
+
+    @property
+    def phase_designation(self):
+        """Calculate phase designation based on occupied circuit positions."""
+        if not (self.breaker_position and self.breaker_poles):
+            return None
+
+        positions = self.get_occupied_positions()
+        if not positions:
+            return None
+
+        # Positions 1,2=A, 3,4=B, 5,6=C, 7,8=A, 9,10=B, 11,12=C, etc.
```

Comment:
Ignorant question - do all power-panels provide three-phase power? Or are there some that only provide A/B?